src/__test__/components/data-exploration/gene-list-tool/FilterGenes.test.jsx

```diff
@@ -17,36 +16,13 @@ describe('FilterGenes', () => {
     expect(search.length).toEqual(1);
   });
 
-  test('sends correct search pattern on search', () => {
+  test('uses correct search pattern on search', () => {
     const mockFilter = jest.fn();
     const component = mount(<FilterGenes onFilter={mockFilter} />);
-    const search = component.find(Search);
-
-    act(() => {
-      search.getElement().props.onSearch('tgf');
-    });
-
-    expect(mockFilter).toHaveBeenCalledTimes(1);
-    expect(mockFilter).toHaveBeenCalledWith({ filterOption: 'Contains', text: 'tgf' });
-  });
-
-  test('change selected search option', () => {
-    const mockFilter = jest.fn();
-    const component = mount(<FilterGenes onFilter={mockFilter} />);
-    const select = component.find(Select);
 
-    act(() => {
-      select.getElement().props.onChange('Starts with');
-    });
-
-    component.update();
-    const search = component.find(Search);
-    act(() => {
-      search.getElement().props.onSearch('tgf');
-    });
+    const searchBox = component.find('.ant-input');
 
-    expect(mockFilter).toHaveBeenCalledTimes(2);
-    expect(mockFilter).toHaveBeenNthCalledWith(1, { filterOption: 'Starts with', text: null });
-    expect(mockFilter).toHaveBeenNthCalledWith(2, { filterOption: 'Starts with', text: 'tgf' });
+    searchBox.simulate('change', { target: { value: 'A' } });
+    expect(searchBox.instance().value).toContain('A');
```

Comment:
What is the point of this test? Inputing a letter and checking if its there after?

Reply:
Yes, it was there checking if the correct string was sent to the api. It is in fact pointless now. I will remove it...

Reply:
maybe you should check if the correct genes are displayed/filtered

Reply:
I have a check for this already

Reply:
I can see it for differential expression but not for the filter genes? Or is it reusing the same component/function?

Reply:
Yes, they are reusing the same component